tcms/bugs/tests/test_views.py

```diff
@@ -68,3 +81,58 @@ def test_reopen_a_closed_bug(self):
         self.assertRedirects(response, redirect_url)
         bug.refresh_from_db()
         self.assertTrue(bug.status)
+
+
+class TestNewBug(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'bugs.add_bug')
+
+        cls.url = reverse('bugs-new')
+
+        cls.summary = 'A shiny new bug!'
+        cls.product = ProductFactory()
+        cls.version = VersionFactory(product=cls.product)
+        cls.build = BuildFactory(product=cls.product)
+        cls.post_data = {
+            'summary': cls.summary,
+            'reporter': cls.tester.pk,
+            'product': cls.product.pk,
+            'version': cls.version.pk,
+            'build': cls.version.pk,
+        }
+
+    def test_get_view(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn('page_title', response.context)
+        self.assertIn('form_post_url', response.context)
+        self.assertEqual(
+            response.context['form'].initial['reporter'].pk, self.tester.pk
+        )
+        self.assertTemplateUsed(response, 'bugs/mutable.html')
```

Comment:
Also assert that product is -1 in the beginning - this is important initial data for the JS code on the page. See `get_form()` method.

Reply:
Still unresolved